irods/path/__init__.py

```diff
@@ -51,22 +49,44 @@ def __new__(cls, *elem_list, **kw):
         absolute_ = kw.pop('absolute',True)
         if kw:
             logging.warning("These iRODSPath options have no effect: %r",kw.items())
-        normalized = cls.resolve_irods_path(*elem_list,**{"absolute":absolute_})
+        normalized = _normalize_iRODS_logical_path(elem_list, absolute_)
         obj = str.__new__(cls,normalized)
         return obj
 
 
-    @staticmethod
-    def resolve_irods_path(*path_elems, **kw):
+def _normalize_iRODS_logical_path(paths, make_absolute):
+
```

Comment:
I think you remove this empty line and this PR should be ready.

Reply:
Done! and #' d